package.json

```diff
@@ -109,15 +109,17 @@
     "serve": "^12.0.1",
     "sharp": "^0.29.1",
     "stylelint": "^14.2.0",
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-recommended": "^6.0.0",
     "tslib": "^2.3.1",
     "typescript": "^4.5.2"
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,mjs}": [
-      "eslint --fix"
+      "lint:js --fix"
```

Comment:
The original way is better, as lint-staged passes the individtual files to the commandline, but this is going to run the full repo

Reply:
Yeah, just realized that as I was testing. Thanks for pointing it out